workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -64,21 +96,35 @@ public static int RunIn(string workingDirectory, string command, params string[]
 
                 var processOutput = new StringBuilder();
 
-                void OnReceived(object sender, DataReceivedEventArgs args)
+                void OnReceived(string output)
```

Comment:
These probably shouldn't be local methods here. As we allocate the string builder regardless of whether we end up using this